lib/beaker/subcommands.rb

```diff
@@ -0,0 +1,22 @@
+require "thor"
+require "fileutils"
+require_relative "shared/subcommands_util"
+
+module Beaker
+  class Subcommands < Thor
+    include Beaker::Shared::SubcommandsUtil
+
+    desc "init", "Initialises the beaker test environment"
+    long_desc <<-LONGDESC
+      Initialises a beaker environment using a combination of environment variables and options
+    LONGDESC
+    option :host, :default => "hosts.yaml", :type => :string, :desc => "The host configuration to test against"
+    option :test, :default => "acceptance/tests/base/", :type => :string, :desc => "The test suite to run"
```

Comment:
have we decided upon a set of flags to pass to beaker for the init phase? i settled on passing --host and --test for now, but adding more flags should be very straightforward